src/commons/crypto/signing/signers/pkcs11/signer.rs

```diff
@@ -452,7 +469,7 @@ impl Pkcs11Signer {
     ) -> Result<UsableServerState, ProbeError<SignerError>> {
         fn slot_label_eq(ctx: &RwLockReadGuard<Pkcs11Context>, slot: Slot, slot_label: &str) -> bool {
             match ctx.get_token_info(slot) {
-                Ok(info) => String::from_utf8_lossy(&info.label).trim_end() == slot_label,
+                Ok(info) => info.label().trim_end() == slot_label,
```

Comment:
It looks to me like this change potentially introduces a new panic, if the underlying token info contains a non-UTF8 compliant value. Does the `label()` call ensure that that is not possible? This change doesn't seem to be related to upgrading cryptoki or did I miss something?